### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by LuBuss**
- Add some docker documentation
- remove admin path
- clean up old files

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,18 +1,11 @@
 .vscode
 node_modules
-public/admin-assets/js/app-*
-public/admin-assets/js/vendor-*
-public/admin-assets/css/bundle-*
-public/admin/index.html
+public/assets/js/app-*
```
Comment (ngoyal16) [inline .gitignore]: `public/assets` can be removed, we are no longer using that path.

File: .gitignore [hunk 2]
```diff
@@ -1,18 +1,11 @@
...
+public/assets/js/vendor-*
+public/assets/css/bundle-*
+public/index.html
 public/content/*
```
Comment (ngoyal16) [inline .gitignore]: `public/content` can be removed, we are no longer using that path.

File: .gitignore [hunk 3]
```diff
@@ -1,18 +1,11 @@
...
 !public/content/*.md
-theme/dist/*
-theme/assets/js/app-*
-theme/assets/js/theme-*
-theme/assets/css/bundle-*
-theme/assets/index.html
-theme/assets/sw.js
-theme/assets/precache-manifest.*.js
 dist/*
 logs/*.log
```
Comment (ngoyal16) [inline .gitignore]: `logs/*` can be removed, we are no longer using that path.

File: config/admin.js [hunk 4]
```diff
@@ -1,8 +1,8 @@
 // config used by dashboard client side only
 module.exports = {
 	// dashboard UI language
-	language: 'en',
+	language: process.env.LANGUAGE || 'en',
 	apiBaseUrl: process.env.API_BASE_URL || 'http://localhost:3001/api/v1',
-	apiWebSocketUrl: process.env.API_WEB_SOCKET_URL || 'ws://localhost:3001',
-	developerMode: true
+	apiWebSocketUrl: process.env.API_WEB_SOCKET_URL, // 'ws://localhost:3001',
```
Comment (ngoyal16) [inline config/admin.js]: fix line number 6 as it contains error

Comment (ngoyal16) [reply config/admin.js]: you should add default ones too

Comment (LuBuss) [reply config/admin.js]: Added default as empty string as docker doesn't allow for empty strings as env.

**Pull request merged**